app/views/devise/registrations/new.html.haml

```diff
@@ -1,34 +1,22 @@
 .container
   .row.justify-content-center
-    .col-md-4
-      %h2.header= t(".sign_up")
-      = form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-        = devise_error_messages!
+    .col-lg-5.col-md-6
+      %h2.header= t('.sign_up')
+      = simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { novalidate: false }) do |f|
         .form-group
-          = f.label :email
-          %br
-          = f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control"
+          = f.input :email, label: 'Email', autofocus: true, autocomplete: 'email', input_html: { class: 'form-control' }, placeholder: 'example@mail.com', required: true
         .form-group
-          = f.label :name, t(".name")
-          %br
-          = f.text_field :name, class: "form-control"
+          = f.input :name, label: t('.name'), input_html: { class: 'form-control', minlength: 3, maxlength: 30 }
         .form-group
-          = f.label :surname, t(".surname")
-          %br
-          = f.text_field :surname, class: "form-control"
+          = f.input :surname, label: t('.surname'), input_html: { class: 'form-control', minlength: 3, maxlength: 30 }
         .form-group
-          = f.label :phone_number, t(".phone_number")
-          %br
-          = f.text_field :phone_number, class: "form-control"
+          = f.input :phone_number, label: t('.phone_number'), input_html: { class: 'form-control' }, placeholder: '+XXXXXXXXXXXX', required: false
```

Comment:
but you agreed to remove a number completely, didn't you?
@shevchukm

Reply:
yes

Reply:
then please remove it everywhere

Reply:
I've done

Reply:
done